Tests/DistributedActorsTests/Cluster/SWIM/Protobuf/SWIM+SerializationTests.swift

```diff
@@ -18,32 +18,34 @@ import SWIM
 import XCTest
 
 final class SWIMSerializationTests: ClusterSystemXCTestCase {
-    func test_serializationOf_ping() throws {
-        let memberProbe = self.testKit.makeTestProbe(expecting: SWIM.Message.self)
-        let ackProbe = self.testKit.makeTestProbe(expecting: SWIM.Message.self)
-        let payload: SWIM.GossipPayload = .membership([.init(peer: memberProbe.ref, status: .alive(incarnation: 0), protocolPeriod: 0)])
-        let ping: SWIM.Message = .remote(.ping(pingOrigin: ackProbe.ref, payload: payload, sequenceNumber: 100))
-        try self.shared_serializationRoundtrip(ping)
-    }
+    func test_serializationOf_ack() async throws {
+        let target = await setUpNode("target") { settings in
+            settings.enabled = true
+        }
 
-    func test_serializationOf_pingRequest() throws {
-        let memberProbe = self.testKit.makeTestProbe(expecting: SWIM.Message.self)
-        let ackProbe = self.testKit.makeTestProbe(expecting: SWIM.Message.self)
-        let payload: SWIM.GossipPayload = .membership([.init(peer: memberProbe.ref, status: .alive(incarnation: 0), protocolPeriod: 0)])
-        let pingReq: SWIM.Message = .remote(.pingRequest(target: memberProbe.ref, pingRequestOrigin: ackProbe.ref, payload: payload, sequenceNumber: 100))
-        try self.shared_serializationRoundtrip(pingReq)
-    }
+        guard let targetSwim = target._cluster?._swimShell else {
+            throw testKit.fail("SWIM shell should be non nil")
+        }
+        let targetID = ActorID(remote: target.settings.uniqueBindNode, path: targetSwim.id.path, incarnation: targetSwim.id.incarnation)
```

Comment:
Have to do this for deserialized id to match

Reply:
Aha I see since we need to force the remote...

There's a `._asRemote` I think that does this